app/javascript/gobierto_investments/webapp/containers/home/Main.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <main class="investments-home-main">
+  <div class="investments-home-main">
```

Comment:
Why so?

Reply:
Because we're already adding the `main` tag in the [template](https://github.com/PopulateTools/gobierto/blob/52b8027626497d80c3839199596fcc9e6f5c80a5/app/views/layouts/application.liquid#L13)

https://dequeuniversity.com/rules/axe/4.2/landmark-no-duplicate-main?application=vue-axe